documents/documents.html

```diff
@@ -17,6 +17,7 @@
     <link rel="stylesheet" href="../components/card.css" />
     <link rel="stylesheet" href="../components/navigation.css" />
     <link rel="stylesheet" href="../components/typography.css" />
+    <link rel="stylesheet" href="../components/snackbar.css" />
```

Comment:
You can create a main stylesheet and import all the stylesheets there. After use the link for main stylesheet in the html code. I believe it would make the code more readable.

Reply:
imported only one link